src/router.js

```diff
@@ -43,7 +44,11 @@ const Router = {
       }
 
       // Disable embedding
-      response.set('Content-Security-Policy', 'frame-ancestors \'none\'');
+      response.set('Content-Security-Policy',
+                   config.get('oauthPostToken') ?
+                     'frame-ancestors filesystem:' :
```

Comment:
I believe `filesystem:` needs single quotes around it: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors

Reply:
Seems not , well with quotes I got an alert that they were ignored... now i am unsure

Reply:
I think you're right. Confusing docs on MDN.

Reply:
Yes I fixed MDN with help of @sheppy (thx)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors$revision/1535837